### DISCUSSION TITLE
twitter功能介紹

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#599] [Show and tell]
https://inheart0317.pixnet.net/blog/post/772339-twitter%E5%8A%9F%E8%83%BD%E4%BB%8B%E7%B4%B9%E5%8F%8A%E6%87%89%E7%94%A8
由於現在很多的行銷都必須更加的國際化，也漸漸發現其實國外更多人在使用的是twitter，而非FB或ig，所以對未來想要將行銷更國際化的話就得了解twitter的一些基本使用。文章從註冊開始講解，非常實用的!

**Comment by hapiwangy** [comment]
以前因為遊戲需要所以辦了twitter但都沒在用，沒想到他有那麼多的好處和使用方式

**Comment by handsomeniko** [comment]
twitter好用，各位紳士懂得都懂

**Comment by Jay-722** [comment]
twitter很好滑阿

**Comment by bloodtrail0511** [comment]
推特主要都是歐美日的人較多，我認識的台灣人比較少有在用的

**Comment by Sazmafuji** [comment]
現在twitter被伊隆·馬斯克買下，twitter應該會有許多改變

**Comment by awkward-willy** [comment]
twitter 在台灣並不流行
前些日子被馬斯克買下的消息曝光後，討論度極高
但最近風波一過好像又沉寂下來了

可是社群軟體還是需要許多周遭的人使用才能發揮最大的效用
看來台灣可能短期內很難形成廣大的 twitter 用戶

**Comment by yenwei12138** [comment]
感覺以台灣人的社交圈來說，twitter沒那麼吸引人，畢竟多數人的同儕都還是在用FB、IG

**Comment by stan890425** [comment]
因為沒甚麼朋友在用twitter，所以在上面不會被朋友騷擾XD

**Comment by Jimmy911218** [comment]
twitter在台灣跟在國外有明顯的使用率差
所以基本在twitter上發言比較少認識的人看比較自在一點

**Comment by Yuhsin000** [comment]
之前有嘗試使用過，但不知道為何就是不太順手

**Comment by Ben6uo** [comment]
twitter原本還好 但至從馬斯克買了之後就超級好用了

**Comment by liangyinglly** [comment]
以前下載twitter 總是被川普洗版

**Comment by leon10456** [comment]
推特的話 各位紳士都能流暢使用吧
